<launch>
<node pkg="agree_esmacat_pkg" name="shared_memory_interface" type="esmacat_ros_interface" output="screen" />
<node pkg="agree_esmacat_pkg" name="keyboard_interface" type="testbed_ros_interface" output="screen" />

<rosparam file="$(find agree_esmacat_pkg)/config/parameters.yaml" />
</launch>
